workflow/search-filter.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Search & Filter
+---
+
+The Search & Filter features serve as the jumping-off point in your data exploration process within Middleware. Data is made available across all contexts within Middleware, allowing you to understand the interconnectedness of your data.
+
+The Filter Bar and Search Fields are two of the simplest methods for sifting through your data.They are both dynamically generated based on the applications, integrations, and infrastructure you have instrumented within Middleware. These features are inclusive (i.e. union set operations) and will display data that satisfies any of the selected filters.
+
+<Note> For more information on log-based search, navigate to the [Log Monitoring - Search & Filter]() document to learn about our temporary log search solution.  </Note>
```

Comment:
If we change the log search and filter doc to be about saving filter combos then this can be removed. We should write this doc with the intent of search / filter to be ubiquitous across the app